src/features/search/components/SearchAutocomplete/SearchAutocomplete.tsx

```diff
@@ -24,8 +24,8 @@ export const SearchAutocomplete: React.FC<Props> = ({ hitComponent: Item, ...pro
       keyboardShouldPersistTaps="handled"
       keyboardDismissMode="on-drag"
       testID="autocompleteList"
-      renderItem={({ item }) => (
-        <Item hit={item as unknown as AlgoliaSuggestionHit} sendEvent={sendEvent} />
+      renderItem={({ item, index }) => (
+        <Item hit={item as unknown as AlgoliaSuggestionHit} sendEvent={sendEvent} index={index} />
```

Comment:
Je suis vraiment pas fan de passer l'index comme ça, je n'aime pas lier de la logique à un index.

Qu'est-ce que tu penses plutôt d'une prop `shouldShowCategory: boolean` que tu passerais de cette façon
```suggestion
      renderItem={({ item, index }) => (
        <Item hit={item as unknown as AlgoliaSuggestionHit} sendEvent={sendEvent} shouldShowCategory={index < 3} />
```

De cette façon tu peux virer le check plus bas `const showCategory = index < 3 && categories.length > 0`

Reply:
Et comme ça, `<Item />` se fiche total d'un index, ce qui l'importe c'est de savoir s'il doit ou non afficher la catégorie :)